crates/bevy_input/src/gamepad.rs

```diff
@@ -638,19 +659,21 @@ impl AxisSettings {
         livezone_upperbound: f32,
         threshold: f32,
     ) -> Result<AxisSettings, AxisSettingsError> {
-        if !(-1.0..=0.0).contains(&livezone_lowerbound) {
+        let within = |value, lower: f32, upper: f32| (lower..=upper).contains(&value);
```

Comment:
I think this should also be moved into a separate refactor PR if that is coming. I personally feel like this is not making the code more readable, but that would be a better place to discuss.